iconrpcserver/utils/json_rpc.py

```diff
@@ -52,14 +52,15 @@ def process_response(self, response, log_extra=None, log_format=None):
             # Log the response before processing it
             self.log_response(response, log_extra, log_format)
             # If it's a json string, parse to object
-            if isinstance(response, basestring):
+            if isinstance(response, str) or isinstance(response, bytes):
```

Comment:
```suggestion
            if isinstance(response, (str, bytes)):
```

Reply:
Thank you for your suggestion!
I applied yours.